infrawatch/service-telemetry-operator
# Import master to stable-1.5

Import changes for STF 1.5.3

- Fixes for 17.0 ir script (#380)
- Move the SNMP trap delivery checks (#381)
- Lock the bundle to OCP v4.10 (#385)
- Make all certs 8yr expiry
- Revert "Make all certs 8yr expiry"
- Make all certs 8yr expiry (#387)
- Expand support for OCP 4.11 (#391)
- Force SGO checkout during build (#388)
- Update oc to 4.11 in jenkins agent (#393)
- Remove OperatorHub.io as a CatalogSource (#394)
- Changes for 4.12 (#401)
- Remove Loki from stf-run-ci (#405)
- GHA checkout action v2 is deprecated (#407)
- Implement SNMPtrap delivery controls (#404)
- Implement changes for operator-sdk-1.26.0 testing (#411)
- Expose ability to set certificate renewal target times (#406)
- Fix default CA cert lifetime values in stf-run-ci (#414)
- fix/client version mismatch (#415)
- Allow oc client version override for Jenkins agent (#416)
- Update CatalogSource for Prometheus Operator deployment (#419)
- Fix path to right collectd service template file for OSP 13 (#418)
- Latest oauth-proxy with bcrypt (#420)
- [jenkins] Add custom context labels for github build status notification (#402)
- Observability Operator for Prometheus & Alertmanager (#421)
- Use stable-v1 cert-manager in CI for OCP >= 4.12 (#424)
- Pinning prometheus version to ease migration to OBO (#425)
- Fix auto-reversion to use_community (#427)
- Fix duplicate alarm expressions (#426)
- [zuul] Create zuul no-op job to test integration (#431)
- Allow bundle deploy from unauth'd repo (#437)
- Manage Operator dependencies with properties.yaml (#423)
- Use the proper parameter for SNMP timeout (#435)
- Support External Elasticsearch (#439)
- Fix bug with supplied user.crt/key not persisting (#442)
- Test with "external" elasticsearch (#441)
- Revert cert-manager channel to "tech-preview" (#444)
- Fix infinite reconcile loop (#445)
- Add checks for not set sto and sgo bundle paths (#447)
- Bump compatibility from 4.10-4.12 to 4.10-4.14 (#448)
- Allow Grafana management in all observability modes (#449)
- Add tests/infrared/17.1 (#440)
- Corrected Hugepages alert summary to match severity (#451)
- [stf-run-ci] Update vars in README (#459)
- Update Dockerfile to create epel repo config when NO_PROXY is undefined (#458)
- Add basic README for OSP 17.1 test deployment (#456)
- Use fully qualified apiVersion on Routes (#453)
- Clean up dependency installation in stf-run-ci (#455)
- Fix static namespace reference (#464)
- [ansible-lint] Use fqcn for modules (#457)
- [stf-run-ci] Update pre-flight checks to be time-limited (#461)
- Expose operator_sdk versions used in CI (#466)
- Remove registry access config tags (#446)
- generate_bundle: add logging (#463)
- Improve logging (#471)
- Relax pod admission controls when using a local catalog index (#450)
- [stf-run-ci] Update pull_secret check for running bundles (#474)
- Install ObO during index-based deployments (#472)
- Update 17.1 helper scripts README (#476)
- [issue#306] Add missing ClusterRoles (#465)
- [stf-run-ci] Use {{ base_dir }} instead of relative paths (#460)
- [stf-run-ci] Update base_dir in setup_stf_local_build.yml (#478)
- [stf-run-ci] Add timeout to validate deployment (#477)
- Adjust the default scrape interval (#454)
- [stf-run-ci] Add requirements.txt (#484)
- Add local generation difference check (#482)
- Bump origin-ansible-operator base image (#487)
- Remove logs API interface from STF (#483)
- Reduce the Ceilometer pollsters in helper script (#489)
- [zuul] Add base job for testing with zuul (#479)
- Move the metrics_result retrieve (#493)
- Increase sensubility smart-gateway ring buffer (#490)
- Disable event pipeline management (#491)
- [zuul] Use include_vars in prepare stage (#496)
- Initial changes for QDR basicAuth (#481)
- [zuul] use 'service-telemetry' as namespace (#497)
- Add var to set QDR auth to none (#498)
- [zuul] Add a job that does a local build and deploys an STF object (#480)
- Bump requests from 2.27.1 to 2.31.0 in /build/stf-run-ci (#486)
- Use vars for registry image container name (#452)
- [stf-run-ci] Generate extra logs if preflight checks fail (#500)
- Bump oauthlib from 3.2.0 to 3.2.2 in /build/stf-run-ci (#485)
- [stf-run-ci][create_catalog] Swap query for a command task (#470)
- Add requires infrastructure annotations (#499)
- [stf-collect-logs] Add a role for log collection (#488)
- [zuul] Add job to deploy from nightly bundles (#494)
- Next release of STF will start at OCP 4.11 (#503)
- Update base image to pass security scans (#502)
- Cluster Observability Operator dependency (#504)
- [better_logging] Show generate_bundle log when task fails (#501)
- Don't deploy events SGs by default (#506)
- Adjust CR annotation to matchdefault scrape interval
- Fix bundle check test (#507)
- Update path to internal TLS CA (#508)
